static/app/views/onboarding/targetedOnboarding/components/firstEventFooter.tsx

```diff
@@ -58,19 +64,28 @@ export default function FirstEventFooter({
     ) {
       return (
         <Button
-          to={`/onboarding/${organization.slug}/mobile-redirect/`}
           priority="primary"
-          onClick={() => {
-            clientState &&
-              client.requestPromise(
-                `/organizations/${organization.slug}/onboarding-continuation-email/`,
-                {
-                  method: 'POST',
-                  data: {
-                    platforms: clientState.selectedPlatforms,
-                  },
-                }
-              );
+          onClick={async () => {
+            if (clientState) {
+              addLoadingMessage(t('Sending you an email to continue onboarding...'));
+              await client
```

Comment:
@Neo-Zhixing why are you using `.then` and `.catch` instead of just doing it in a try/catch block? When it's functionally equivalent we should use the try/catch block